src/LanguageModel/OpenMP.h

```diff
@@ -110,4 +110,11 @@ inline bool isDebugOutlined(const llvm::StringRef& funcName) { return funcName.s
 
 inline bool isGetThreadNum(const llvm::StringRef& funcName) { return funcName.equals("omp_get_thread_num"); }
 
+inline bool isGetThreadNumGuardStart(const llvm::StringRef& funcName) {
+  return funcName.equals("omp_get_thread_num_guard_start");
+}
+inline bool isGetThreadNumGuardEnd(const llvm::StringRef& funcName) {
+  return funcName.equals("omp_get_thread_num_guard_end");
+}
```

Comment:
Because these strings are used in other files, they can be made into named constants a referred to from preprocessing as well.

```suggestion
// Custom named functions we insert during pre-processing to mark guarded regions
const std::string OpenMPThreadGuardStart{"omp_get_thread_num_guard_start"};
const std::string OpenMPThreadGuardEnd{"omp_get_thread_num_guard_end"};

inline bool isGetThreadNumGuardStart(const llvm::StringRef& funcName) {
  return funcName.equals(OpenMPThreadGuardStart);
}
inline bool isGetThreadNumGuardEnd(const llvm::StringRef& funcName) { return funcName.equals(OpenMPThreadGuardEnd); }
```